src/controllers/home-page.controller.ts

```diff
@@ -17,6 +17,7 @@ export class HomePageController {
   constructor(
     @inject(PackageKey) private pkg: PackageInfo,
     @inject(RestBindings.Http.RESPONSE) private response: Response,
+    @inject(RestBindings.HTTPS_OPTIONS) private config: RestServerConfig,
```

Comment:
So ... we are binding the RestConfig to `HTTPS_OPTIONS` ... and I tried to re-bind the config back to `RestBindings.CONFIG` but that threw an error since that key contains a `#` (it's accessing `ApplicationConfig#rest`). So not sure what we should change in `@loopback/rest` ... hopefully this starts the discussion around it.

Reply:
RestBindings.CONFIG cannot be used for binding as it's a property of `ApplicationConfig`.